src/tests/ftest/control/daos_system_query.py

```diff
@@ -0,0 +1,47 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+from daos_utils import DaosCommand
+
+
+class DaosSystemQuery(TestWithServers):
+    """Test Class Description:
+    Test the 'daos system query' command. This command requests client-relevant information
+    from the DAOS system.
+
+    :avocado: recursive
+    """
+
+    def test_system_query(self):
+        """JIRA ID: DAOS-7983
+        Test Description: Test daos system query.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=control,daos_cmd
+        :avocado: tags=daos_system_query,test_daos_system_query
+        """
+        daos_cmd = DaosCommand(self.bin)
```

Comment:
We have a helper function for this
```suggestion
        daos_cmd = self.get_daos_command()
```

Reply:
Fixed